models/category.js

```diff
@@ -16,13 +16,31 @@ module.exports = (sequelize, DataTypes) => {
     {
       Name: DataTypes.STRING,
       IconBackImg: DataTypes.STRING,
+      IconBackImgUrl: {
+        type: DataTypes.VIRTUAL,
+        get() {
+          return this.assetUrl('IconBackImg');
+        },
+      },
       NavBackImg: DataTypes.STRING,
+      NavBackImgUrl: {
+        type: DataTypes.VIRTUAL,
+        get() {
+          return this.assetUrl('NavBackImg');
+        },
+      },
```

Comment:
Here in the model, we add new virtual attributes to return the full image url for the attached files.